Dockerfiles/mlpack-codespace/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM mcr.microsoft.com/devcontainers/universal:focal
+
+LABEL maintainer="james.balamuta@gmail.com"
+
+## For apt to be noninteractive.
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+## Install baseline dependencies not found in the devcontainer
+## For details on what's included, please see:
+## https://github.com/devcontainers/images/tree/main/src/universal
+RUN apt-get update -qq && apt-get install -y --no-install-recommends \
+    binutils-dev \ 
+    txt2man \
+    doxygen \
+    liblapack-dev \
+    libblas-dev \
+    libarpack2 \
+    libsuperlu-dev \
+    libstb-dev && \
```

Comment:
I think we will also need `libcereal-dev`, unless that's already installed. :+1:

Reply:
Added